### PR TITLE
[WIP] Codacy fixes

### PR DESCRIPTION
**Pull request by JavaEria**

### CODE REVIEW

File: event/models.py [hunk 1]
```diff
@@ -5,8 +5,8 @@ class Event(models.Model):
 
     title = models.CharField(max_length=255)
     description = models.TextField()
-    start_datetime = models.DateTimeField()
```
Comment (mashhoodr) [inline event/models.py]: can we please revert this change? We will keep underscores in python @JavaEria @taleymirza

Comment (JavaEria) [reply event/models.py]: Okay

**Comment by taleymirza** [comment]
@mashhoodr I have researched about the security issue of regex 
 the current regex is  /^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/
safe-regex is a package that detect possibly catastrophic, exponential-time regular expressions  detecting our regex to be unsafe. when I analyze the regex I didn't found any bug in it then I search for the same problems over the net I found[ this ](https://github.com/substack/safe-regex/issues/10) so we can change {3} into 3 times (?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)) to resolve this issue

**Comment by mashhoodr** [comment]
@taleymirza thanks for the review. lets leave this issue since there doesnt seem to be specific danger from it at the moment

File: client/src/containers/ListEvents.js [hunk 2]
```diff
@@ -37,16 +37,16 @@ class ListEvents extends Component {
         dispatch(userLogout());
     }
 
-    handleSubmit = event => {
+    handleSubmit = (event) => {
         event.preventDefault();
 
-        const { title, description, start_datetime, end_datetime, organisation } = event.nativeEvent.target.elements;
+        const { title, description, startDateTime, endDateTime, organisation } = event.nativeEvent.target.elements;
```
Comment (mashhoodr) [inline client/src/containers/ListEvents.js]: @JavaEria this will break at the moment

**Comment by mashhoodr** [comment]
@rahil051 please review and merge if working

**Comment by JavaEria** [comment]
There is 1 failed check for having double quotes instead of single quotes, should we change that as well?

**Comment by mashhoodr** [comment]
no we will be using single quotes

**Comment by JavaEria** [comment]
The app is working fine as well.

**Comment by mashhoodr** [comment]
@rahil051 please review and merge

**Comment by rahil051** [comment]
@mashhoodr can checks for single quotes be ignored on codacy ?

**Comment by mashhoodr** [comment]
will look into that
